tools/depends/target/python27/Makefile

```diff
@@ -14,6 +14,7 @@ endif
 ifeq ($(OS),ios)
   HOSTPLATFORM=_PYTHON_HOST_PLATFORM="darwin"
   LINK_ICONV=-framework CoreFoundation -liconv
+  CFLAGS+=-DDISABLE_PROCESS_CREATION
```

Comment:
Since pythons configure doesn't pick it up and we provide it explicitley into the configure environment i can do so - but python is at fault anyway in this case ...